src/app.js

```diff
@@ -1 +1,28 @@
 'use strict';
+/* eslint-disable no-console */
+
+const fs = require('fs');
+
+function copyFile() {
+  const [fileToCopy, fileToCreate] = process.argv.slice(2);
+
+  if (fileToCopy === fileToCreate) {
+    return;
+  }
```

Comment:
As [Oleksii Andriushyn](https://github.com/Orpheus29) mentioned the `copyFile` function shouldn't know anything about args in the terminal. It should just copy the files.